app/views/mission_control/jobs/jobs/_general_information.html.erb

```diff
@@ -34,6 +34,12 @@
       </td>
     </tr>
   <% end %>
+  <tr>
+    <th>Retries</th>
+    <td>
+      <%= job.respond_to?(:arguments) ? job.arguments.first["executions"] : 0 %>
```

Comment:
In which case `job` doesn't respond to `arguments`? 🤔 

I think this should be always an instance of `ActiveJob::JobProxy`, so perhaps we should make this available there.

Reply:
@rosa Perfect!
I put it in JobProxy ;)